src/accessibility/RovingTabIndex.tsx

```diff
@@ -270,9 +270,11 @@ export const RovingTabIndexProvider: React.FC<IProps> = ({
 // onFocus should be called when the index gained focus in any manner
 // isActive should be used to set tabIndex in a manner such as `tabIndex={isActive ? 0 : -1}`
 // ref should be passed to a DOM node which will be used for DOM compareDocumentPosition
-export const useRovingTabIndex = (inputRef?: Ref): [FocusHandler, boolean, Ref] => {
+export const useRovingTabIndex = <T extends HTMLElement, >(
```

Comment:
```suggestion
export const useRovingTabIndex = <T extends HTMLElement>(
```

Reply:
hmmm, I swore this gave syntax errors before, https://stackoverflow.com/a/56989122/796832

I did just update everything after installing the latest Node.js 14.18.2 🤷

Reply:
If you didn't have an `extends` condition then yes, but that makes it non-ambiguous vs JSX

Reply:
Ahh, yes! I was previously trying to use just generic `T` before which led to too much refactoring across the app and re-thought about it to use `extends`. Thanks for explaining the intricacies here 😊

Reply:
Yeah its a bit horrid, `<T = HTMLElement>` as a default generic wouldn't be enough for JSX to be happy about it, but `extends` is 🤷